dts/arm/silabs/efr32xg13p.dtsi

```diff
@@ -181,6 +181,11 @@
 		};
 	};
 
+	bt_hci_silabs: bt_hci_silabs {
+		compatible = "silabs,bt-hci";
+		status = "disabled";
+	};
```

Comment:
As mentioned by chat, in the `efr32xg13` series, only `efr32mg13` and `efr32bg13` support bluetooth.

Reply:
I believe this declaration should move to `dts/arm/silabs/efr32bg13p632f512gm48.dtsi`.